restructure.yml

```diff
@@ -160,3 +161,8 @@ topics:
     # Disable deduplication
     deduplication:
       enable: false
+  questionnaire_response:
+    # Specify an alternative path format.
+    pathProperties:
+      format: ${projectId}/${userId}/${topic}/${value:name}/${filename}
+      plugins: fixed value
```

Comment:
does this mean we use `fixed` and `value` plugin by default?

Reply:
The default is `fixed time key value`, but for this path only a subset of the plugins is used.